concept/thing/thing_manager.rs

```diff
@@ -632,31 +575,28 @@ impl ThingManager {
 
     fn attribute_instance_range_from_attribute_type_range(
```

Comment:
This whole method doesn't need to exist anymore - you should pass 
`KeyRange<AttributeType<'static>>,` instead, see the  `get_has_reverse_by_attribute_and_owner_type_range` method